content/en/blog/_posts/2021-08-16-minreadysecond-statefulsets.md

```diff
@@ -0,0 +1,48 @@
+---
+layout: blog
+title: 'minReadySeconds for StatefulSets'
+date: 2021-08-27
```

Comment:
It's handy to update the path to the Markdown file to match. Hugo doesn't care but us humans can get confused.